docs/DeveloperGuide.md

```diff
@@ -151,6 +151,11 @@ How the architecture interacts with each other [DIAGRAM]
 The UI is a public class that consists of **three components** that is made up `Scanner`, `Constants` 
 and `Print Methods`. 
 
+![archi](./images/UiClassDiagram.png)
```

Comment:
Perhaps some attributes or methods can be omitted as it may not do much in explaining the UI Component
![스크린샷 2021-04-01 오전 9 23 46](https://user-images.githubusercontent.com/42374564/113230618-f74d8080-92cb-11eb-8a3b-aea3706eb624.png)